test/setup_bundler_test.rb

```diff
@@ -869,29 +869,6 @@ def test_only_returns_environment_if_bundle_was_composed_ahead_of_time
     end
   end
 
-  def test_ignores_bundle_bin
```

Comment:
Based on what I was able to gather, setting `BUNDLE_BIN`​ no longer automatically creates binstubs when running `bundle install`​. You need to run `bundle binstubs --all`​ to achieve that same behaviour.

I had trouble finding a related pull request, but after investigating the code, this is what I think happened.